apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -509,10 +509,9 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     function canVoteOnBehalfOf(uint256 _voteId, address[] _voters, address _representative) external view returns (bool) {
         Vote storage vote_ = _getVote(_voteId);
 
-        if (!_canRepresentativesVote(vote_)) {
+        if (!_canRepresentativesVote(vote_) || _voters.length > MAX_VOTES_DELEGATION_SET_LENGTH) {
```

Comment:
Interesting, do you prefer to not revert (it does save some space 😄)?

Reply:
You changed my mind, let's revert :)

Reply:
Addressed in beacbd92213037ea26201ea834229ed5f960432b